srcpkgs/hashcash/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'hashcash'
+pkgname=hashcash
+version=1.23
+revision=1
+archs="~*-musl"
+build_style=gnu-makefile
+make_build_args="LIBCRYPTO=$XBPS_CROSS_BASE/usr/lib/libcrypto.a generic-openssl docs"
+makedepends="libressl-devel"
+depends="openssl>=3.0"
```

Comment:
I don't think we have `openssl` for now.

Reply:
@sgn If I understand correctly, libressl provides openssl>=3 API, so if we have a dependency on openssl, then both openssl and libressl implementations work.

Reply:
OK, libressl provides `openssl-$version`. I think we can just write `libressl` there. Or `openssl`.

Reply:
Does this command call the openssl executable? Otherwise this shouldn't be here. Also you can't depend on the openssl package while it doesn't exist.

Reply:
@sgn @ericonr OK, I renamed to `libressl>=3.0`

Reply:
No, no version specification. And I don't think this calls the libressl executable, so why is it added as a dependency?

Reply:
@ericonr according to manual, it should be package name, not executable binary: https://github.com/void-linux/void-packages/blob/master/Manual.md#optional-variables and since I can be sure it's working with libressl 3d version I added >= 3.0. Do you think I should remove version constraint?

Reply:
The only relevant thing in the `libressl` package is the `openssl` executable, equivalent to what an `openssl` package would provide. Since this program doesn't call that executable, why is it a dependency?

Reply:
`xbps-src` will correctly detect the right package to add to `depends`, so this is unnecessary.

Reply:
(and the correct package is `libcryptoXX`, not `libressl`.

Reply:
@abenson @ericonr OK, so I've removed the runtime dependency line, is it good now? edad5fb1071b140a26bbadc2806ccdb95bb467c5